io/disk/ssd/nvmetest.py

```diff
@@ -431,3 +433,10 @@ def testsubsystemreset(self):
         cmd = '%s subsystem-reset %s' % (self.binary, self.device)
         if process.system(cmd, ignore_status=True, shell=True):
             self.fail("Subsystem reset failed")
+
+    def test_create_namespaces(self):
+        """
+        creates the specified number of namespaces on nvme drive
+        """
+        device = self.device.split("/")[-1]
+        nvme.create_n_namespaces(device, self.ns_count)
```

Comment:
@Naresh-ibm please  rename the create_n_namespace to proper one which is in utils